include/valkey/cluster.h

```diff
@@ -289,17 +292,6 @@ valkeyClusterAsyncContext *valkeyClusterAsyncConnectWithOptions(const valkeyClus
 void valkeyClusterAsyncDisconnect(valkeyClusterAsyncContext *acc);
 void valkeyClusterAsyncFree(valkeyClusterAsyncContext *acc);
 
-/* Initiate and connect as separate steps. */
-valkeyClusterAsyncContext *valkeyClusterAsyncContextInit(const valkeyClusterOptions *options);
-int valkeyClusterAsyncConnect(valkeyClusterAsyncContext *acc);
-
-/* Callback option configurable after a context initiation, enabling that the
- * valkeyClusterAsyncContext pointer can be given as privdata in the callback. */
-int valkeyClusterAsyncSetEventCallback(valkeyClusterAsyncContext *acc,
```

Comment:
Mention in the migration guide that these are deleted?

Reply:
Added to the removed section in migration guide.